remove.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Initialize basic vars and helpers
+source blueprint-lib/init.sh
+
+##
+# Add dependencies here
+##
+
+# source blueprint-lib/docker.sh
+
+##
+# ADD BLUEPRINT CODE BELOW HERE
+#
+# BASE_PATH is the full path to the project root
+# APP_NAME is the name of the Django app that will be modified
+##
+
+BLUEPRINT="Messenger"
+NAME="Messenger"
```

Comment:
You should remove duplicate variable and update on `human_name`.

Reply:
blueprint name may be different from human name

Reply:
just keep it as template, it should not be a matter